snafu/benchmarks/_load_benchmarks.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""Holds function for automagically importing all benchmark modules in ``snafu.benchmarks``"""
+import os
+import pkgutil
+import importlib
+import logging
+from typing import Tuple, List
+
+
+def load_benchmarks(logger: logging.Logger = None) -> Tuple[List[str], List[str]]:
```

Comment:
I feel like this code should be part of the `benchmarks` package's `__init__.py` and be done implicity when built. That way you don't have to manually call this function further upstream

Reply:
That's super smart, I like that. I think I might keep the code separate, just to keep stuff organized, but I'll add an implicit call.